perception/mil_mlp/README.md

```diff
@@ -0,0 +1,168 @@
+# Transfer Learning Object Detection - A Short Tutorial
+
+# @TODO
+1. Configurable image download
+2. Add docker build to the install script. Until then docker build must be done manually.
+3. Have datasets stored on internal server so it can be downloaded.
+4. Configurable resize (Low importance due to no significant performance gain)
+5. Configurable CSV input (Currently it still assumes you are handing it a json or PASCAL so it will error out if
+those are not provided.)
+6. Add python dependencies to the install script.
```

Comment:
This should happen before merging. Developers should be able to run the code in our repo after running install script

Reply:
Only caveat is that right now we have to build a custom tensorflow image, which implies that the user would need to download the full tensorflow repo and run their custom script to generate a docker image. Unfortunately these images are going to be tied to the specific GPU you are running.